frontend/src/components/jobs/result/utils/sections.jsx

```diff
@@ -84,6 +104,18 @@ export function JobActionsBar({ job }) {
           titlePlacement="top"
         />
       )}
+      <IconButton
+        Icon={MdOutlineRefresh}
+        onClick={() => {
+          createJob(formValues).then(refetch);
+        }}
+        color="light"
+        size="xs"
+        title="run"
```

Comment:
the title could be more "descriptive". Something like "Retry the analysis" or "Force run the same analysis" would be better from a user perspective

Reply:
Then, even if the button is cool, as you may notice, there is another possible user experience problem.

The button "retry" is right next the "delete" one. They are both very little and it could be easy to click the wrong one. At this time I would suggest to make those 2 buttons larger and add a brief description in both of them (like the other ones on the right). In that way chances of wrong clicks are highly reduced and all the buttons follow the same rendering. What do you think? Could you please make this last change?

Reply:
ah, another last thing. IT would be cool if, once the button is clicked, the user would be redirected to the job result page of the new analysis. Otherwise, as you can see in your video, the user should do few more clicks

Reply:
yes, these changes would make the UX better, working on them.

Reply:
review on this @mlodic ? 
<img width="577" alt="Screenshot 2023-02-01 at 7 54 35 PM" src="https://user-images.githubusercontent.com/90976669/216069626-06a65a91-3de5-49e1-904e-3efb424a3274.png">

ig now the button's are well described, and the size's good to, very hard to misclick now.

Reply:
Cool but you just suggested me another change, if you would like to do that :P 

I guess it could make sense that for smaller screens the description is not shown :P Could you do that we the already available react-strap libraries? In that way, once there is no space, the page would simply adapt to the size of the screen.

Reply:
@mlodic i believe that the smaller screens are more in need of that description, for a better a UX and less misclicks in all, removing them might defeat the purpose.

Reply:
also for the navigation part, since im leveraging the `createJob` component, in case of multiple observables it creates multiple jobs and navigates back to the `jobs` page, improves UX or go about navigating to the actual job itself?

https://user-images.githubusercontent.com/90976669/216088602-24ab88af-8bf7-430e-9738-9fa2cccf2104.mov

would like to know if there's a better way to implement this.

Reply:
ok for the description.

I didn't understand your last question, could you rewrite that please?

Reply:
yes had a doubt, it just resolved itself.